src/crimson/osd/replicated_recovery_backend.cc

```diff
@@ -892,6 +892,86 @@ ReplicatedRecoveryBackend::trim_pushed_data(
   return {intervals_usable, data_usable};
 }
 
+seastar::future<hobject_t>
+ReplicatedRecoveryBackend::prep_push_target(
+  const ObjectRecoveryInfo &recovery_info,
```

Comment:
nit:
```suggestion
   const ObjectRecoveryInfo& recovery_info,
```

The alternative is to switch `*` and `&&`.